lib/gcloud/storage/bucket.rb

```diff
@@ -433,9 +446,9 @@ def files prefix: nil, delimiter: nil, token: nil, max: nil, versions: nil
       #   file = bucket.file "path/to/my-file.ext"
       #   puts file.name
       #
-      def file path, generation: nil
+      def file path, generation: nil, key: nil, key_sha256: nil
```

Comment:
@blowmage? (My personal preference is for long, explicit names that hold closely to the underlying API...)